### PR TITLE
Release/v1.5.0

### PR DESCRIPTION
**Pull request by kshrutik**
Release notes:

- New remediation jobs added for AWS:

> - Enable encryption for Cloudtrail logs (aws_cloudtrail_logs_encrypted) Rule ID: 5c8c25e47a550e1fb6560bac
> - Enables KMS automated key rotation (aws_kms_key_rotates) Rule ID: 5c8c26217a550e1fb6560c12
> - Remove Cloudtrail S3 Bucket Public Access (aws_s3_cloudtrail_public_access) Rule ID: 5c8c265d7a550e1fb6560c65

- Updated remediation jobs for AWS:
> - Enable S3 Access Logging job is updated to encrypt the target S3 Bucket and has an updated Readme

- Added list of supported remediation jobs to the Readme

### CODE REVIEW

**Review by nandeshguru — DISMISSED** [review]

File: remediation_worker/jobs/aws_s3_cloudtrail_public_access/aws_s3_cloudtrail_public_access.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import logging
+import sys
+import boto3
+
+logging.basicConfig(level=logging.INFO)
+
+
+class CloudtrailS3RemovePublicAcces:
+    def parse(self, payload):
+        """Parse payload received from Remediation Service.
```
Comment (bhanu13) [inline remediation_worker/jobs/aws_s3_cloudtrail_public_access/aws_s3_cloudtrail_public_access.py:27]: Nit: Missing an 's' in the end for access `CloudtrailS3RemovePublicAccess`

File: remediation_worker/jobs/s3_enable_access_logging/README.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 
 ##### Rule ID:
-5c8c265e7a550e1fb6560c67
+* 5c8c265e7a550e1fb6560c67
+* 5c8c265c7a550e1fb6560c63
 
 ##### Rule Name:
-S3 access logging is not enabled
+* S3 access logging is not enabled
+* Access logging is not enabled for CloudTrail S3 buckets
```
Comment (bhanu13) [inline remediation_worker/jobs/s3_enable_access_logging/README.md:13]: Nice!

File: remediation_worker/jobs/s3_enable_access_logging/README.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
 # Enable S3 Access Logging
 
-This job enables access logging for an S3 bucket.
+This job enable Access Logging for an S3 Bucket. It checks for a target bucket with name "vss-logging-target-{cloud_account_id}-{region}", if the bucket exists then it gives log delivery permissions to that bucket and then stores the logs into it. If the bucket does not exist then it creates one.
```
Comment (bhanu13) [inline remediation_worker/jobs/s3_enable_access_logging/README.md:3]: Nit: enables

**Review by nandeshguru — DISMISSED** [review]

**Review by bhanu13 — APPROVED** [review]

**Review by vikramsinghvirdi — APPROVED** [review]
LGTM

**Review by mngaonkar — APPROVED** [review]

**Review by vikramsinghvirdi — APPROVED** [review]

**Review by nandeshguru — APPROVED** [review]

**Review by bhanu13 — APPROVED** [review]

**Pull request merged**